cohort_4/playbook/ACTIVITIES.md

```diff
@@ -88,6 +89,109 @@
 
 <!-- markdown-toc end -->
 
+# Creating a safe space
+
+The one rule that we have at Hack Camp is that we make it a safe space.
+We all know how bad it feels when we are made fun of or when someone makes
+us feel uncomfortable.
+
+Below are some scenarios which may occur and how to handle them:
```

Comment:
Perhaps make the below scenario part of this higher-level headline of creating a safe space?